Runtime/Scripts/EdgeEventsManager.cs

```diff
@@ -102,11 +99,19 @@ private void OnApplicationPause(bool pause)
 
     private void OnApplicationQuit()
     {
+      if (fcThread.FCPerformanceThread.IsAlive)
+      {
+        fcThread.FCPerformanceThread.Interrupt();
```

Comment:
One easy thing you could do is have a class function that interrupts the thread as part of the FCPerformanceThreadClass. That way you don't have to expose the Thread variable and you can rename FCPerformanceThreadClass back to FCPerformanceThread.

Then from here you can just call `fcThread.Interrupt()` and the class function might look like:

```
class FCPerformanceThread
{
    Thread thread;
    string hostOverride;
    uint portOverride;
.
.
.
    internal void Interrupt() {
        if (thread != null)
        {
             thread.Interrupt();
        }
    }
.
.
.
```

Reply:
Added